### PR TITLE
Remove version from docker-compose project name

### PR DESCRIPTION
**Pull request by exekias**
Latest docker-compose version includes `-` in the project name, while
previous versions were striping it. This is causing tests to fail as containers
are no longer in the same network as beats. (beats is launched by local docker-compose
which is in an old version, while testing containers are launched by the new version)

I'm removing the version part of the current name to ensure we support both old and
new docker-compose behaviors, the commit hash is there already anyway.

This PR is also pinning existing python requirements to avoid new
versions breaking tests.

### CODE REVIEW

File: libbeat/tests/system/requirements.txt [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
-autopep8
-six
-docker-compose
+autopep8==1.3.5
+backports.ssl-match-hostname==3.5.0.1
+cached-property==1.4.2
+certifi==2018.1.18
+chardet==3.0.4
+docker==3.2.1
+docker-compose==1.21.0
```
Comment (jsoriano) [inline libbeat/tests/system/requirements.txt]: What about updating also the version of docker-compose in `.travis.yml`?

**Comment by ruflin** [comment]
@exekias Will have to backport this to 6.2 too?

**Comment by exekias** [comment]
We can backport it to be sure, although I think it's not affected as the version there doesn't contain `-`

**Comment by jsoriano** [comment]
We could pin versions in 6.2 in any case.

File: libbeat/tests/system/requirements.txt (hunk 1 above)
Comment (exekias) [reply libbeat/tests/system/requirements.txt]: Done, in theory it should not affect the result, but :+1: to use the same one there

**Comment by exekias** [comment]
jenkins, test it again please

**Pull request merged**